package.json

```
@@ -38,7 +38,7 @@
     "@edx/frontend-platform": "8.1.1",
     "@edx/reactifex": "^2.1.1",
     "@openedx/frontend-build": "14.1.2",
-    "@openedx/paragon": "22.7.0",
+    "@openedx/paragon": "22.8.0",
```

Should this mirror StudioFooter's dependency?
```suggestion
    "@openedx/paragon": ">= 21.11.3 < 23.0.0",
```

---

Good catch @pomegranited!
The version was wrong in both!
We need paragon 22.8.0 or upper here, so it should be `^22.8.0`. We don't need the `< 23.0.0` part because the `^` would not allow updates to major versions.